kt-extensions/src/test/java/com/mindera/skeletoid/kt/extensions/utils/CollectionTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.mindera.skeletoid.kt.extensions.utils
+
+import org.junit.Assert
+import org.junit.Test
+
+class CollectionTest {
```

Comment:
CollectionExtenstionsTest?

Reply:
Same thing here, really — `Collection.kt`

Do you feel like we should rename that class to `CollectionExtensions.kt`